.travis.yml

```diff
@@ -6,8 +6,8 @@ before_script:
   - npm install -g truffle@4.1.14 ganache-cli@6.1.8
   - npm install
 
-script: 
-  - ganache-cli -m=plasma_mvp > /dev/null &
+script:
+  - ganache-cli -m "they only media any modify banner suffer pole tag rule creek harvest" > /dev/null &
```

Comment:
what is this?

Reply:
so i included tests that verify the inputs you gave me. i.e. the challenges. so i need to hard code in the sender's private key. 

if you don't think these types of tests are necessary, i'll remove them

Reply:
do we have to use the hardcoded values?